projects/driver_controls/src/stm32f0xx/ads1015.c

```diff
@@ -163,6 +163,9 @@ StatusCode ads1015_read_raw(Ads1015Storage *storage, Ads1015Channel channel, int
     return status_code(STATUS_CODE_INVALID_ARGS);
   }
   *reading = storage->channel_readings[channel];
```

Comment:
Generally we'd like to fail earlier to minimize changes that we make.

Reply:
we will need the change either way tho because even if the channel is disabled we want to read int max.

Reply:
My point is why do we want that behavior? The user isn't going to check the reading if it's invalid, so what's the point?

Reply:
I thought it would be a good thing to have on the screen at least when I was testing it all this time.
So you re saying the user better not even check a disabled channel right? If he does he will get garbage because the read function wouldnt change *reading passed to it.

Reply:
Right, I think it was good for validating your code and we were able to find a problem using it, but for actual use, reading a disabled channel is an invalid case.